pkg/deploy/identity-provider/deployment_keycloak.go

```diff
@@ -100,7 +101,7 @@ func GetSpecKeycloakDeployment(
 		}
 	}
 
-	cmResourceVersions := deploy.GetAdditionalCACertsConfigMapVersion(deployContext)
+	cmResourceVersions := certificates.GetAdditionalCACertsConfigMapVersion(deployContext)
```

Comment:
Why do we need the config map version here?

Reply:
because we mount certificates into keycloak pod as well